metals/src/main/scala/scala/meta/internal/metals/WorkspaceLspService.scala

```diff
@@ -626,6 +632,11 @@ class WorkspaceLspService(
           .getOrElse(onNotFound())
       )
 
+  private def executeRunDoctor() = {
```

Comment:
We can [ust the function in the doctor itself to make it possible to just pass WorkspaceDoctor around